### PR TITLE
OpenSSH_6.8 changed the output of "ssh-keygen -l"

### PR DESCRIPTION
**Pull request by lloydde**
Fixes DOCKER-324.

### CODE REVIEW

File: tools/sdc-docker-setup.sh [hunk 1]
```diff
@@ -109,6 +109,21 @@ function dockerInfo
 }
 
 
+# Return ssh fingerprint in the form "he:xh:ex:he:xh..."
+# OpenSSH_6.8 changes the -l flag output.
+function sshGetMD5Fingerprint() {
+    local s
+    sshPubKeyPath=$1
```
Comment (trentm) [inline tools/sdc-docker-setup.sh]: should use `local` too I believe

File: tools/sdc-docker-setup.sh [hunk 2]
```diff
@@ -109,6 +109,21 @@ function dockerInfo
...
+    s=$(ssh-keygen -E md5 -l -f "$sshPubKeyPath" 2> /dev/null)
+    if [[ $? -eq  0 ]]; then
+        echo "$s" | awk '{print $2}' | tr -d '\n' | cut -c 5-;
```
Comment (trentm) [inline tools/sdc-docker-setup.sh]: is `| cut -c 5-;` accurate? Doesn't it use a ':' delimiter?  If so, would prefer `cut -d: -f2-`. Fine either way tho. I'm being pedantic. :)

**Comment by trentm** [comment]
otherwise, lgtm!

**Comment by lloydde** [comment]
Merged in 729e5898ffa91bd14736e192529a467fce7bef31.

**Pull request closed without merging**